features/methods.feature

```diff
@@ -10,7 +10,7 @@ Feature: Methods catalogue tests
     Scenario: Check for dropdown content
         Given I am on the methods catalogue page
         When I click on the collapsible drop down
-        Then I see the dropdown content "Methods are sorted in hierarchical order with expert groups as the top level. For example, within an expert group are different themes, and within the themes are different methods. To find out more about expert groups and themes, please visit the analysis function website (opens in a new window)."
+        Then I see the dropdown content "Methods are sorted in hierarchical order with expert groups as the top level. For example, within an expert group are different themes, and within the themes are different methods. To find out more about expert groups and themes, please visit the analysis function website."
```

Comment:
Was the "(opens in a new window)" part accidentally added part of the SPP-9550 ticket last week? And now being removed? If so all good by me =) I was getting confused if this was in the main branch from the start and now I figured after some git digging and looking at the SPP-9550 ticket. Unless I am wrong and something else is going on. Thanks =)

Reply:
Yes it was added as the goal of that ticket, but now I know how to use the visually hidden class I thought it would be best to change this. The browser still displays "(opens in a new window)" within the link (I tested with a screen reader) but it is hidden from sighted users. I had to modify the test to remove this though as the way Jinja strips out the span tags was causing formatting issues. I changed the methods.py test file to remove the "(opens in a new window)" string plus a few extra whitespace characters that don't get removed by Jinja. Let me know if there's a better way of doing this, e.g. if I should keep the new window text in the test or not.